src/main/java/org/eclipse/emfcloud/jackson/annotations/JsonAnnotations.java

```diff
@@ -147,6 +148,18 @@ public static String getIdentityProperty(final EClassifier classifier) {
       return getValue(classifier, "JsonIdentity", "property");
    }
 
+   /**
+    * Returns {@code true}, if the feature is annotated to be treated as raw JSON.
+    *
+    * @param feature any feature
+    * @return {@code true}, if raw (de)serialization should be done for this feature
+    */
+   public static boolean isRawValue(final EStructuralFeature feature) {
```

Comment:
Please update the year in the copyright header to 2022, i.e., 2019-2022.

Reply:
Done.